node/middlewares/generateMiddlewares/groupEntries.ts

```diff
@@ -30,7 +31,7 @@ const groupEntityEntries = async (entity: string, files: string[], bucket: strin
       newFiles.push(entry)
       await vbase.saveJSON<SitemapEntry>(bucket, entry, {
         lastUpdated: currentDate(),
-        routes: currentRoutes,
+        routes: uniq(currentRoutes),
```

Comment:
```suggestion
        routes: uniqBy(i => i, currentRoutes),
```

`uniq` is really bad because it uses ramda's equal. This might not be a problem now but eventually can become a monster.
I recommend change all the usage of `uniq` to `uniqBy`